InternxtDesktop/Views/Backups/BackupConfigView.swift

```diff
@@ -121,10 +121,12 @@ struct BackupConfigView: View {
                 self.changeFolders()
             } else {
                 try await self.backupsService.startBackup(onProgress: {progress in })
+                EventsUtils.trackSuccessBackup(foldersToBackup: backupsService.foldersToBackup.count)
             }
             
         } catch {
             self.showErrorDialog(message: "BACKUP_ERROR_BACKING_UP")
+            EventsUtils.trackFailureBackup(error: error, foldersToBackup: backupsService.foldersToBackup.count)
```

Comment:
Track this event from the XPCBackupService, this is a good place:

https://github.com/internxt/drive-desktop-macos/blob/main/XPCBackupService/XPCBackupService.swift#L100

Reply:
its done